administrator/components/com_media/Controller/Api.php

```diff
@@ -108,16 +107,16 @@ public function files()
 					// Grab options
 					$options = array();
 					$options['url'] = $this->input->getBool('url', false);
-
 					$data = $this->getModel()->getFiles($adapter, $path, $this->input->getWord('filter'), $options);
-
 					break;
+
```

Comment:
it is easy for anyone for read the file, if we have it like

`case:
      break;
case:
      break;`

it seems harder for read them because of everything is together